### ISSUE TITLE
Some LC475 crashes in setlong() function.

### ISSUE THREAD

**Issue by Mustermann68k** [Phipli/Soft-475-Overclock#1]
It seems that setlong() function work for some LC475 and crashes at others, manly if Frequency was changed to 33Mhz by resistor swap.

**Comment by Phipli** [comment]
My investigations so far - on one of my 475s the computer hangs when setlong() runs during boot with the logic board resistors set to either 25 or 33MHz (the only unusual thing is that I could only find a 330ohm resistor when I set it back to 25MHz, but 330 vs 300 will not make a difference). On my other 475, it always sets to 37.5MHz when I use setlong(). Other people online have had no such issues see https://mastodon.social/@danieltufvesson/112973044546446785 ). I have not confirmed if people with fully working setups have a 343S0161 (like me) or a 343S1135 clock generator. This difference needs testing.

More confusingly, your (@Mustermann68k) program works fine, when run from the desktop.

My thoughts...

- Is it a compatibility issue due to the code running early during boot?
- Is it a timing issue, this would explain the variation in behaviour between scenarios. Do we need some NOPs?
- Is it a hardware difference? Clock generator? LC vs full 040? Amount of RAM, amount of VRAM? Speed of RAM / VRAM?
- Is it the current video resolution?
- Is it OS version?
- Have I passed an incorrect variable type by accident?

I have already tested to see if loading the extension later during boot (by changing the type to "INIT") made a difference and it didn't.

I have already tested several OS versions (7.5.3 and 8.1) and there was no change.

I need to do a little more debugging to see where in setlong it crashes. It being an extension makes it slightly more difficult to debug.
